src/vaadin-radio-button.ts

```diff
@@ -4,6 +4,20 @@ import { RadioButtonMixin } from './vaadin-radio-button-mixin';
 
 /**
  * `<vaadin-radio-button>` is a Web Component for radio buttons.
+ *
+ * @attr label-empty - Set when there is no label provided.
+ * @attr disabled - Set when the radio button is disabled.
+ * @attr focus-ring - Set when the radio button is focused using the keyboard.
+ * @attr focused - Set when the radio button is focused.
+ * @attr checked - Set when the radio button is checked.
+ *
+ * @slot - Slot used to place the label.
```

Comment:
I think this would be a bit more clear
```suggestion
 * @slot - Slot for the label content.
```

"Slot used to place the label" could be interpreted to mean that you should place a `<label>` into it.
Not a blocker.

Reply:
@Haprog thanks, fixed. Also rebased the PR.

Reply:
Thanks. Looks good, approved.